irteus/test/test-triangulation.l

```diff
@@ -41,18 +41,11 @@
   (let ((plate (make-cube 1000 1000 5))
         (params ;; params for holes (list (list segments radius center) .. )
          (list
-          #| ;; failed even after ...
```

Comment:
test should always monotonic increases, so how about test two cases. with `params1` and `params2`

Reply:
Thank you. I fixed https://github.com/euslisp/jskeus/pull/585/commits/ffe00eaf32ecdd836d1aaeab1054a49257bf052a